app/src/main/AndroidManifest.xml

```diff
@@ -14,17 +14,34 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
+    <activity
+      android:name=".testing.NavigationDrawerTestActivity"
+      android:screenOrientation="portrait"
+      android:theme="@style/OppiaThemeWithoutActionBar">
+      <intent-filter>
```

Comment:
SplashActivity should be the launcher activity.

Reply:
Done.